Acrolinx.Sidebar/Acrolinx.Sidebar.csproj

```diff
@@ -151,6 +151,18 @@
       <HintPath>..\packages\log4net.2.0.17\lib\net45\log4net.dll</HintPath>
       <Private>True</Private>
     </Reference>
+    <Reference Include="Microsoft.Web.WebView2.Core, Version=1.0.2592.51, Culture=neutral, PublicKeyToken=2a8ab48044d2601e">

```

Comment:
@Thilo-Acrolinx Wondering why dependabot has added the section, instead of just modifying the version numbers